src/torchmetrics/functional/image/ssim.py

```diff
@@ -468,10 +462,10 @@ def multiscale_structural_similarity_index_measure(
 
     Example:
         >>> from torchmetrics.functional import multiscale_structural_similarity_index_measure
-        >>> preds = torch.rand([1, 1, 256, 256], generator=torch.manual_seed(42))
+        >>> preds = torch.rand([3, 3, 256, 256], generator=torch.manual_seed(42))
         >>> target = preds * 0.75
-        >>> multiscale_structural_similarity_index_measure(preds, target)
-        tensor(0.9558)
+        >>> multiscale_structural_similarity_index_measure(preds, target, data_range=1.0)
```

Comment:
why can't we stick to `None` here? that seems to be the most intuitive one for me.

Reply:
Actually I think `None` is the least intuitive since it estimates the min and max from the data. 

If this was real data (image for example) you know what the theoretical lower bound and upper bound on the values are and you wouldnt want to estimate them from the data. Otherwise you might get different ranges over different batches and the SSIM values wouldn't be directly comparable.

It's counterintuitive because who would want that?